ext/nnstreamer/tensor_filter/tensor_filter_llamacpp.cc

```diff
@@ -133,7 +133,10 @@ TensorFilterLlamaCpp::configure_instance (const GstTensorFilterProperties *prop)
   }
 
   /** load dynamic backends */
+#if !defined(__ANDROID__)
   ggml_backend_load_all ();
```

Comment:
Android does not allow application to access `/` directory.
`ggml_backend_load_all` accesses root directory to scan multiple backend so permission error occurs.

Reply:
can you check below PR?
https://github.com/ggerganov/llama.cpp/pull/10812

Reply:
It has already been fixed recently :)
Thanks for letting me know
https://github.com/ggerganov/llama.cpp/releases/tag/b4321